assets/pages/bundles-landing/components/countrySwitcherHeaderContainer.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+
+import CountrySwitcherHeader from 'components/headers/countrySwitcherHeader/countrySwitcherHeader';
+
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { State } from '../bundlesLandingReducers';
+
+
+const availableCountriesGroups: CountryGroupId[] =
+  ['GBPCountries', 'UnitedStates'];
+
+// ----- Functions ----- //
+
+function handleCountryGroupChange(value: string): void {
+  switch (value) {
+    case 'UnitedStates':
+      window.location.href = '/us';
+      break;
+    case 'AUDCountries':
+      window.location.href = '/au';
+      break;
+    case 'EURCountries':
+      window.location.href = '/eu';
+      break;
+    default:
+  }
```

Comment:
Doesn't this lose the querystring? Do we want `window.location.pathname` instead?

Reply:
Fixed